compiler/src/dotty/tools/dotc/typer/Inliner.scala

```diff
@@ -1335,6 +1357,16 @@ class Inliner(call: tpd.Tree, rhsToInline: tpd.Tree)(using Context) {
     /** Suppress further inlining if this inline typer has already issued errors */
     override def suppressInline(using Context) =
       ctx.reporter.errorCount > initialErrorCount || super.suppressInline
+
+    private def inlineIfIsNestedInlineCall(tree: Tree)(using Context): Tree =
+      if
+        !suppressInline &&
```

Comment:
Cosmetic: I think it reads better of the `&&` comes first on each subsequent line

Reply:
Done